test/test_automated_install.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from textwrap import dedent
+
+@pytest.fixture
+def Pihole(Docker):
+    ''' generates stubs of files that normally execute code '''
+    ''' (moved to Dockerfile in some cases) '''
+    return Docker
+
+SETUPVARS = {
+    'piholeInterface' : 'eth99',
+    'IPv4addr' : '192.168.100.2',
+    'piholeIPv6' : 'True',
+    'piholeDNS1' : '4.2.2.1',
+    'piholeDNS2' : '4.2.2.2'
+}
+
+def test_setupVars_are_sourced_to_global_scope(Pihole):
```

Comment:
The actual 2 tests are here, basically run the Docker I stuffed all the bash scripts into earlier (and stubbed the install during image creation) and assert things work like we expect.